client/components/SingleProduct.js

```diff
@@ -1,18 +1,52 @@
 import React, {Component, Fragment} from 'react'
 import {connect} from 'react-redux'
+import {Link} from 'react-router-dom'
 import {selectProduct} from '../store/products'
 import {getToCart} from '../store/cart'
-import ProductReview from '../components/ProductReview'
+import {fetchReviews} from '../store/reviews'
+
+import {
+  withStyles,
+  Typography,
+  Card,
+  CardContent,
+  Button
+} from '@material-ui/core/styles'
+import StarRatings from 'react-star-ratings'
+
+const styles = {
```

Comment:
The styles being done in here instead of in an external css file is killing me. Also the only thing being used in the import above is withStyles, so why import the rest?